src/config/containers_and_vms/podman.md

```diff
@@ -0,0 +1,33 @@
+# Podman
+
+**Podman** is a daemonless engine for developing, managing, and running pods,
+containers, and container images.
+
+## Installation
+
+Install the `podman` package.
+
+## Checking installation
+
+To check your installation of podman, run the following command:
+
+`$ podman run -dt -p 8080:8080/tcp registry.fedoraproject.org/f29/httpd`
```

Comment:
I think it's better to use code block instead of inline code?

```
    ```sh
    $ podman run -dt ....
    ```
```

Reply:
Okay, will change it